<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manageRecipe</class>
 <widget class="QWidget" name="manageRecipe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="RecipeListWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>811</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>279</y>
     <width>811</width>
     <height>208</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="RecipeName">
        <property name="text">
         <string>Recipe Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Instructions">
        <property name="text">
         <string>Instructions</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Quantity</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTextEdit" name="recipeName"/>
      </item>
      <item>
       <widget class="QTextEdit" name="instructions"/>
      </item>
      <item>
       <widget class="QTextEdit" name="quantity"/>
      </item>
      <item>
       <widget class="QPushButton" name="addRecipe">
        <property name="text">
         <string>AddRecipe</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>169</y>
     <width>811</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTableWidget" name="groceryListWidget"/>
    </item>
   </layout>
  </widget>
  <zorder>RecipeListWidget</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>verticalLayoutWidget_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
